examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -108,6 +108,7 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
       onTap: () async {
         await showCupertinoModalPopup<void>(
           context: context,
+          semanticsDismissible: true,
```

Comment:
Maybe we can just document this on https://master-api.flutter.dev/flutter/cupertino/CupertinoDatePicker-class.html. It already says that it can be shown with showCupertinoModalPopup and we just add that you'll have to set this to true.

Reply:
done